src/inference/include/openvino/runtime/intel_gna/properties.hpp

```diff
@@ -0,0 +1,223 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief A header for advanced hardware related properties for GNA plugin
+ *        To use in set_property() and get_property() methods of plugins
+ *
+ * @file config.hpp
+ */
+#pragma once
+
+#include "openvino/runtime/properties.hpp"
+
+namespace ov {
+namespace intel_gna {
+
+/**
+ * @brief Property to get an std::string of GNA Library version, usually in the form
+ * <API_REVISION>.<RELEASE_LINE>.<RELEASE>.<BUILD>
+ */
+static constexpr Property<std::string, PropertyMutability::RO> library_full_version{"GNA_LIBRARY_FULL_VERSION"};
```

Comment:
can it be a structure with int fields?

Reply:
@ilya-lavrenov we have no agreement for that between plugin and gna lib team. I think it is not safe. @dorloff please comment

Reply:
https://github.com/intel/gna/blob/main/src/gna-lib/CMakeLists.txt#L13

Reply:
I did not have an explicit confirmation from the library folks that this format will always be like this, so would suggest to keep it as one string.